eclipse-che/che-server
# Return the default resolver if the factory URL refers directly to a devfile

### What does this PR do?
Check if the factory URL is a raw devfile URL. If so, use the default resolver,  which resolves factories from a direct URL to a devfile content.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://issues.redhat.com/browse/CRW-4366

### How to test this PR?

1. Deploy Che using the PR image: `chectl server:deploy -p=openshift -i=quay.io/ivinokur/che-server:next`
2. Start a workspace from a gitlab / bitbucket RAW devfile URL.
See workspace starts successfully.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
